lib/Transport.js

```diff
@@ -5,6 +5,7 @@ var Transport = module.exports = function(path, auth) {
 };
 
 var request = require("superagent");
+var Perry = require("perry");
```

Comment:
there is nothing wrong with this module if used the for unique features it provides, which in this case is exactly what's needed.